Xamarin.Forms.Platform.iOS/PlatformRenderer.cs

```diff
@@ -126,21 +51,14 @@ public override bool ShouldAutorotateToInterfaceOrientation(UIInterfaceOrientati
 
 		public override void ViewDidAppear(bool animated)
 		{
-			Platform.DidAppear();
 			base.ViewDidAppear(animated);
+			Platform.DidAppear();
```

Comment:
`Platform.DidAppear` isn't doing anything magical to require to be called before the base call. In most cases, base should appear first.